source/tutorial/control-access-to-mongodb-with-kerberos-authentication.txt

```diff
@@ -21,16 +21,38 @@ authentication protocol for large client/server systems.
 Prerequisites
 -------------
 
+.. include:: /includes/fact-confirm-enterprise-binaries.rst
+
+.. include:: /includes/fact-kerberos-FQDN-repica-sets.rst
+
 Setting up and configuring a Kerberos deployment is beyond the scope of
-this document. This tutorial assumes you have configured a
+this document. Please refer to the `MIT Kerberos documentation
+<https://web.mit.edu/kerberos/krb5-latest/doc/>`_ or documentation
+for your operating system for information on how to configure a
```

Comment:
maybe tweak to your operating system documentation (so that we don't have 2 for phrases).

Either that or I can merge this and tweak for you.  Let me know which you prefer.

Reply:
I trust your decision :)

Reply:
Then I'll merge and tweak tomorrow and publish.  Thanks as always!